src/include/gurt/shm_alloc.h

```diff
@@ -0,0 +1,134 @@
+/**
+ * (C) Copyright 2024 Intel Corporation.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause-Patent
+ */
+
+#ifndef __DAOS_SHM_ALLOC_H__
+#define __DAOS_SHM_ALLOC_H__
+
+#include <stdint.h>
```

Comment:
it's probably better to create 1 header for this rather than multiple public headers that users of this module need to always include.

Reply:
Thank you very much! Just fixed it. Now only necessary APIs are exposed in this header file.